### PR TITLE
Added jsconfig json file to fix navigation and autocomplete in cypress tests

### PR DESCRIPTION
**Pull request by Prasannams97**
Fixes: #721

### CODE REVIEW

File: cypress-tests/cypress/jsconfig.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+  "compilerOptions": {
+    "baseUrl": ".",
+    "paths": {
+      "Texts/*": ["./constants/texts/*"],
+      "Selectors/*": ["./constants/selectors/*"]
+    }
+  }
+}
```
Comment (yedhink) [inline cypress-tests/cypress/jsconfig.json:9]: You shoud include cypress modules for auto-completion.

Similary set allowJs to true

Refer the jsconfig part in https://docs.cypress.io/guides/tooling/IDE-integration#Set-up-in-your-Dev-Environment

**Review by yedhink — COMMENTED** [review]
@Prasannams97 _a

Please go through the comment.

File: cypress-tests/cypress/jsconfig.json (hunk 1 above)
Comment (Prasannams97) [reply cypress-tests/cypress/jsconfig.json]: Updated.

**Comment by Prasannams97** [comment]
@yedhink _a Please check.

**Comment by yedhink** [comment]
@Prasannams97 _a

Why did you remove the alias paths? Isn't it needed?

If it's not needed, then you can assign this PR back to me.

**Comment by Prasannams97** [comment]
> @Prasannams97 _a
> 
> Why did you remove the alias paths? Isn't it needed?
> 
> If it's not needed, then you can assign this PR back to me.

Added alias paths

**Comment by Prasannams97** [comment]
@yedhink _a Please check.

**Pull request merged**